sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructType.scala

```diff
@@ -104,6 +104,13 @@ case class StructType(fields: Array[StructField]) extends DataType with Seq[Stru
   /** Returns all field names in an array. */
   def fieldNames: Array[String] = fields.map(_.name)
 
+  /**
+   * Returns all field names in an array. This is an alias of `fieldNames`.
+   *
+   * @since 2.3.0
```

Comment:
This is too late to be merged to 2.3.0. Please change it to 2.4.0.

Reply:
+1

Reply:
Yup, I was thinking about it too.